src/screen.rs

```diff
@@ -832,6 +832,7 @@ impl Screen {
         // Output the left prompt if it has changed.
         if left_prompt != zelf.actual_left_prompt {
             zelf.r#move(0, 0);
+            zelf.write_mbs_if_some(&term.and_then(|term| term.prompt_start.as_ref()));
             let mut start = 0;
             for line_break in left_prompt_layout.line_breaks {
                 zelf.write_str(&left_prompt[start..line_break]);
```

Comment:
> run bind \eg 'echo; git status; git remote -v; echo; commandline -f repaint'

right, we offer bindings like `\el` and `\w` so this should definitely be supported